db/pf-schema-X.Y.Z.sql

```diff
@@ -122,6 +122,12 @@ INSERT INTO `node_category` (category_id,name,notes) VALUES ("2","guest","Guests
 INSERT INTO `node_category` (category_id,name,notes) VALUES ("3","gaming","Gaming devices");
 
 --
+-- Insert 'voice' category
+--
+
+INSERT INTO `node_category` (category_id,name,notes) VALUES ("4","voice","VoIP devices");
```

Comment:
The answer is simple. Because that is the way it was always done.
It was that way since node_category was introduced in PacketFence 1.9.0.

The bigger question is do we have any code that depends on those roles being set to those category_ids.

If not the I would recommend to remove it.
Else if we do then we create an issue and address in the next release.